front/src/modules/simulationResult/components/ChartHelpers/ChartHelpers.ts

```diff
@@ -307,6 +363,32 @@ export const interpolateOnTime = <
   return specificInterpolateOnTime(dataSimulation, keyValues, listValues);
 };
 
+export const interpolateOnTimeV2 = <
```

Comment:
Same for interpolateOnTimeV2 and interpolateOnTime

Reply:
Good catch, done.